src/Library/Config/YamlConfig.php

```diff
@@ -0,0 +1,42 @@
+<?php declare(strict_types=1);
+
+namespace SandwaveIo\Office365\Library\Config;
+
+final class YamlConfig
+{
+    /**
+     * @var array<string, array<string>>
+     */
+    private array $yaml;
+
+    /**
+     * YamlConfig constructor.
+     * @param array<string, array<string>> $yaml
+     */
+    public function __construct(array $yaml)
+    {
+        $this->yaml = $yaml;
+    }
+
+    public function getClassName(): ?string
+    {
+        $className = array_key_first($this->yaml);
+
+        if ($className !== null) {
+            return $className;
+        }
+
+        return null;
```

Comment:
```suggestion
        return array_key_first($this->yaml);
```

I don't understand the original version. If it is not null return it and otherwise return null. My suggestion is exactly the same, right?

Reply:
let's see, array_key_first can return int|string|null but i got some complaints from phpstan. let me check